src/test/java/stringcalculator/CalculatorNumbersTest.java

```diff
@@ -0,0 +1,27 @@
+package stringcalculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CalculatorNumbersTest {
+
+    @ParameterizedTest
+    @DisplayName(value = "컴마나 콜론을 붙이면 숫자의 합을 반환한다.")
+    @CsvSource(value = {"1,2,3|6","1:3:5|9"},delimiter = '|')
+    void addValueTest(String calculateForm, int result) {
+        CalculatorNumbers calculatorNumbers = CalculatorNumbers.createNumbers(calculateForm);
+        assertThat(calculatorNumbers.sum()).isEqualTo(result);
+    }
```

Comment:
테스트 코드도 유지보수의 대상이기 때문에 조금 더 가독성을 신경써주면 좋을 것 같습니다. 😃
테스트 케이스가 적은 경우에는 상관이 없지만 서비스가 성장해나감에 따라서 테스트 코드도 자연스럽게 많아질텐데요. 
이런 경우에는 지금처럼 테스트 코드를 작성하게되면 테스트하려는 부분이 어떤 것인지 파악하기가 조금 어려워질 것 같아요. 

가독성을 높이는데 개인적으로 given, when, then의 형식으로 테스트 코드를 짜는 것을 추천드립니다. 😃 

```suggestion
    void addValueTest(String calculateForm, int expected) {
        CalculatorNumbers sut = CalculatorNumbers.createNumbers(calculateForm);
        
        var actual = sut.sum();
        
        assertThat(actual).isEqualTo(expected);
    }
```

Reply:
```suggestion
 @ParameterizedTest
    @DisplayName(value = "컴마나 콜론을 붙이면 숫자의 합을 반환한다.")
    @CsvSource(value = {"1,2,3|6","1:3:5|9"},delimiter = '|')
    void addValueTest(String calculateForm, int result) {
        assertThat(CalculatorNumbers.createNumbers(calculateForm).sum()).isEqualTo(result);
    }

    @Test
    @DisplayName(value = "커스텀 구분자를 사용해서 숫자의 합을 반환한다.")
    void customAddValueTest() {
        assertThat(CalculatorNumbers.createNumbers("//&\n3&4&5").sum()).isEqualTo(12);
    }
```

@Rok93 
좋은 의견 감사합니다 :) given when then은 확실히 말씀하신대로 단계 파악 및 가독성에 도움이 될 것 같아서 주신 말씀이라고 생각합니다. 말씀 주신 가독성도 일리 있는 얘기입니다. 
사실 이부분에 대해서는 여러 의견을 들었고, 저는 이부분에 대해서 조금 다른 생각이 있는데요, 
DisplayName으로 어느정도 어떤 테스트인지 파악할 수 있겠다는 생각이 들었습니다.
그리고 given when then도 좋은 구조이나 테스트코드에서 불필요한 객체 생성을 줄임으로서 길이도 줄이고 불필요하게 메모리를 할당받지 않으려고 극단적으로 줄여보았습니다.
어떻게 생각하실까요? 😄